rom/dev/README.md

```diff
@@ -235,13 +235,31 @@ The following flows are conducted exclusively when the ROM is operating in ACTIV
 The following flows are conducted when the ROM is operating in the manufacturing mode, indicated by a value of `DEVICE_MANUFACTURING` (0x1) in the `CPTRA_SECURITY_STATE` register `device_lifecycle` bits.
 
 #### UDS Provisioning
```

Comment:
We should clarify that this flow is optional. An integration can still choose to generate UDS in it's own way.

Reply:
**One correction:** An integration can still choose to generate UDS in it's own way **"only in passive mode"**

Reply:
Updated.